test.sml

```diff
@@ -7,7 +7,7 @@ use "test/lexer/comments.sml";
 use "test/lexer/strings.sml";
 use "test/lexer/smoke.sml";*)
 
-use "test/parser/smoke.sml";
```

Comment:
Can we add back in the actual smoke test for this and the lexer. To run against the whole fixtures dir?